### PR TITLE
fix(kafka): log a warning if the kafka health check fails

### PR DESCRIPTION
**Pull request by huebner-j**

### CODE REVIEW

File: sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/health/KafkaHealthCheck.java [hunk 1]
```diff
@@ -31,6 +31,7 @@ protected Result check() {
     try (AdminClient adminClient = AdminClient.create(KafkaProperties.forAdminClient(config))) {
       adminClient.listTopics().names().get(2, TimeUnit.SECONDS);
     } catch (Exception e) {
+      logger.warn("Kafka health check failed", e);
```
Comment (JoergSiebahn) [inline sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/health/KafkaHealthCheck.java:34]: We should not use the logger from `AdminClientConfig` but an own logger for this class.

Comment (huebner-j) [reply sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/health/KafkaHealthCheck.java:34]: oh yes, you are absolutely right. Thanks for your review.

**Comment by JoergSiebahn** [comment]
@SDA-SE/sda-commons-maintainer should we postpone this PR until #303 is merged?

**Comment by Fox32** [comment]
I'm not exactly sure if we want to log that. I remember that we already removed such log messages because they are to chatty. Maybe just put the exception into the message itself?

**Comment by christopher-cudennec** [comment]
> @SDA-SE/sda-commons-maintainer should we postpone this PR until #303 is merged?

+1 

New features should be added after DW2 migration was successful. Looks like #303 will be merged soon, doesn't it?

**Comment by dhenneke** [comment]
I think we could already merge this. But for that, please change the base to `feat/dropwizard2` branch and rebase the commit. The master branch is read-only until #303 will be merged (which is very soon hopefully)!

**Comment by dhenneke** [comment]
We will release the fix when we release DW2 since we currently don’t plan to do a further pre-release. I hope it is not too urgent.

**Comment by christopher-cudennec** [comment]
Thanks @dhenneke . I think that's totally fine! :)

**Pull request merged**